apps/condo/domains/meter/hooks/useImporterFunction.ts

```diff
@@ -137,6 +156,14 @@ export const useImporterFunctions = (): [Columns, RowNormalizer, RowValidator, O
         }
         addons.meterResourceId = METER_RESOURCE_ABBREVIATION_TO_ID[String(meterResourceTypeAbbr)]
 
+        let parsedReadingSubmissionDate = dayjs(readingSubmissionDate, DATE_PARSING_FORMAT)
+        if (!parsedReadingSubmissionDate.isValid()) {
+            parsedReadingSubmissionDate = dayjs(readingSubmissionDate, MONTH_PARSING_FORMAT)
+        }
+        if (parsedReadingSubmissionDate.isValid()) {
```

Comment:
It looks like a runtime error, but I’m not sure

Reply:
No, it does not produces a runtime error.
In case of incorrect value, it will be handled and pushed to errors array of current row in `meterReadingValidator`.